seatunnel-engine/seatunnel-engine-client/src/test/resources/streaming_fakesource_to_file_complex.conf

```diff
@@ -20,7 +20,7 @@
 
 env {
   parallelism = 1
-  job.mode = "BATCH"
```

Comment:
and why chnage the job mode?

Reply:
Because I saw that in the relevant use cases, this one is a streaming one and the batch engine does not support checkpoints.

https://github.com/apache/seatunnel/blob/a19720ccf67b2e3ef66a217e841dba12ccadb28e/seatunnel-engine/seatunnel-engine-client/src/test/java/org/apache/seatunnel/engine/client/MultipleTableJobConfigParserTest.java#L88